torchci/lib/jobUtils.ts

```diff
@@ -235,6 +235,10 @@ export function removeJobNameSuffix(
 }
 
 export async function hasS3Log(job: RecentWorkflowsData): Promise<boolean> {
+  if (job.logUrl !== undefined && job.logUrl !== "") {
+    const res = await fetch(job.logUrl, { method: "HEAD" });
+    return res.status !== 404;
```

Comment:
why such a specific status code check? Why not check if the status is a success vs failure code?

Reply:
The original code later checks for 404, so I was just copying it